src/back/mod.rs

```diff
@@ -311,3 +311,12 @@ impl crate::TypeInner {
         }
     }
 }
+
+impl std::fmt::Display for crate::SwitchCaseValue {
```

Comment:
I can move the implementation elsewhere but it's useful since all backends use the C-like syntax for their integer literals, except spirv which is a binary representation.